app/controllers/api/v2/hosts_controller.rb

```diff
@@ -339,6 +339,20 @@ def template
         end
       end
 
+      api :GET, "/hosts/:id/templates", N_("Get provisioning templates for the host")
+      param :id, :identifier_dottable, :required => true
```

Comment:
I don't know what this is, can you please add a description? :slightly_smiling_face:

Reply:
Not sure what more information I can add to this as `Get provisioning templates for the host` is all this does. its based on:
https://github.com/theforeman/foreman/blob/develop/app/controllers/hosts_controller.rb#L892

Reply:
What does `identifier_dottable` mean?

Reply:
That was the part that was confusing me

Reply:
Thats the param type for host id (there are more uses of it in this controller), if users open the apipie docs its explained there
![Screenshot from 2022-09-22 22-10-49](https://user-images.githubusercontent.com/30431079/191841749-093342bc-700d-4fc8-a625-e71d43b9017b.png)

Reply:
Gotcha. thanks!